script.cu.lrclyrics/lib/gui.py

```diff
@@ -67,7 +67,7 @@ def get_settings(self):
         for scraper in os.listdir(LYRIC_SCRAPER_DIR):
             # meh to python3 creating folders
             if os.path.isdir(os.path.join(LYRIC_SCRAPER_DIR, scraper)) and scraper != '__pycache__' and ADDON.getSettingBool(scraper):
-                exec ('from lib.culrcscrapers.%s import lyricsScraper as lyricsScraper_%s' % (scraper, scraper))
+                exec ('from lib.culrcscrapers.%s import lyricsScraper as lyricsScraper_%s' % (scraper, scraper), globals())
```

Comment:
I ran into the same issue with my add-on. I moved to `importlib.import_module` that just returns you an object of the reference module from which you can just use the classes?

Reply:
@basrieter i'm afraid i have no idea how to use importlib.
unless you think my simple solution is wrong, i'd prefer to stick with import.